client/src/layout/SearchBox.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { IconButton, InputBase } from "@material-ui/core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
```

Comment:
You can use Material UI's built in icon library instead of importing a separate one `@material-ui/icons`

Reply:
Hey Dan, so we decided to use font awesome for now since we didn't install matieral-ui/icons either.